<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>param_plot_widget</class>
 <widget class="QWidget" name="param_plot_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Realizations</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="realizationScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>356</width>
        <height>363</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Parameters</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="parametersScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>356</width>
        <height>363</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="plotPushButton">
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
